File: backend/src/main/java/store/andefi/utility/SortBy.java
package store.andefi.utility;

public enum SortBy {
  LATEST("latest"),
  RATING_ASC("rating_asc"),
  RATING_DESC("rating_desc");

  public static final String DEFAULT_SORT_BY = "latest";

  private final String value;

  SortBy(String value) {
    this.value = value;
  }

  @Override
  public String toString() {
    return this.value;
  }
}

File: frontend/app/src/main/java/store/andefi/utility/SortBy.kt
package store.andefi.utility

enum class SortBy(private val value: String) {
    LATEST("latest"),
    RATING_ASC("rating_asc"),
    RATING_DESC("rating_desc");

    override fun toString(): String {
        return this.value
    }

    companion object {
        const val DEFAULT_SORT_BY: String = "latest"
    }
}
